test/index.js

```diff
@@ -73,9 +73,32 @@ describe('sequentially promises', () => {
 
     Promise.resolve(
       promiseq(promises)
-      /* catching errors will force the .then of the wrapper promise to be
-         called after the last promise is executed */
-      .catch(() => {})
+      .catch(() => {
+        /*
+          This catch block exists in order to force the '.then' of the wrapper promise
```

Comment:
There is no way to "force a then block to execute" so this statement is false.
Too long of a description.  Be pithy and get to the point.